src/Sulu/Bundle/AutomationBundle/Resources/public/js/main.js

```diff
@@ -9,7 +9,9 @@
 
 require.config({
     paths: {
-        suluautomation: '../../suluautomation/js'
+        suluautomation: '../../suluautomation/js',
+
+        'services/suluautomation/task-manager': '../../suluautomation/js/services/task-manager'
```

Comment:
Is that really necessary? I thought that's only required when we have the a validation type, where the path doesn't match.